### ISSUE TITLE
[BUG] Error when running autotrain CLI from huggingface -  unrecognized arguments

### ISSUE THREAD

**Issue by thinkingkoala** [huggingface/autotrain-advanced#647]
### Prerequisites

- [X] I have read the [documentation](https://hf.co/docs/autotrain).
- [X] I have checked other issues for similar problems.

### Backend

Hugging Face Space/Endpoints

### Interface Used

CLI

### CLI Command

subprocess_command = ["autotrain", "spacerunner", "--project-name", slugged_lora_name, "--script-path", spacerunner_folder, "--username", username, "--token", token, "--backend", "spaces-a10gs", "--env",f"HF_TOKEN={token};HF_HUB_ENABLE_HF_TRANSFER=1", "--args", spacerunner_args]

AutoTrain advanced CLI: error: unrecognized arguments: --args pretrained_model_name_or_path=stabilityai/stable-diffusion-xl-base-1.0;pretrained_vae_model_name_or_path=madebyollin/sdxl-vae-fp16-fix;instance_prompt=A photo of TOK;dataset_name=./a6a7f02e-f9cf-4878-aa2e-358dd3a26870;caption_column=prompt;output_dir=test1805;mixed_precision=bf16;resolution=1024;train_batch_size=2;repeats=3;gradient_accumulation_steps=1;learning_rate=1.0;text_encoder_lr=1.0;adam_beta1=0.9;adam_beta2=0.999;optimizer=prodigy;train_text_encoder_ti_frac=0.5;lr_scheduler=constant;lr_warmup_steps=0;rank=32;max_train_steps=500;checkpointing_steps=100000;seed=42;prior_loss_weight=1.0;num_new_tokens_per_abstraction=2;num_train_epochs=1;adam_weight_decay=0.0001;adam_epsilon=1e-08;prodigy_decouple=True;prodigy_use_bias_correction=True;prodigy_safeguard_warmup=True;max_grad_norm=1.0;lr_num_cycles=1;lr_power=1.0;dataloader_num_workers=0;local_rank=-1;cache_latents;gradient_checkpointing;train_text_encoder_ti;with_prior_preservation;class_prompt=a photo of a person;num_class_images=150;class_data_dir=72d4e0ef-1513-46ec-8e7a-8ae7166a3353

### Error Logs

![image](https://github.com/huggingface/autotrain-advanced/assets/139218006/16f2de77-4d37-4ff4-bf84-0a50159746ba)

### Additional Information

This error appeared just a couple of days ago when running autotrain from this Hugging Face space https://huggingface.co/spaces/multimodalart/lora-ease
Can't make it work from my private space either.

Would appreciate it if you take a look

**Comment by abhishekkrthakur** [comment]
it was fixed in lora-ease. closing the issue :)
